pints/_log_likelihoods.py

```diff
@@ -14,26 +14,41 @@
 
 class AR1LogLikelihood(pints.ProblemLogLikelihood):
     r"""
-    Calculates a log-likelihood assuming AR1 noise model
+    Calculates a log-likelihood assuming AR(1) (autoregressive order 1)
+    errors. In this error model, the ith error term
```

Comment:
Can you leave a blank line after the first sentence?
The general format should be:
```
One line explanation.

More stuff if needed.
```
The one-line explanation is used by IDEs etc. to give a quick idea of what the method does